<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="softTab">
         <attribute name="title">
          <string>Софт</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="spacing">
           <number>9</number>
          </property>
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea">
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="softArea">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>520</width>
               <height>163</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="softLayout">
              <property name="leftMargin">
               <number>9</number>
              </property>
              <property name="topMargin">
               <number>9</number>
              </property>
              <property name="rightMargin">
               <number>9</number>
              </property>
              <property name="bottomMargin">
               <number>9</number>
              </property>
              <property name="spacing">
               <number>2</number>
              </property>
              <item row="1" column="0">
               <widget class="QLabel" name="noSoftLabel">
                <property name="text">
                 <string>Нет программ для установки </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QProgressBar" name="softProgressBar">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="textVisible">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="invertedAppearance">
                 <bool>false</bool>
                </property>
                <property name="textDirection">
                 <enum>QProgressBar::TopToBottom</enum>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="updateTab">
         <attribute name="title">
          <string>Обновления</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QScrollArea" name="scrollArea_2">
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="updateArea">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>520</width>
               <height>163</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="updateLayout">
              <property name="spacing">
               <number>2</number>
              </property>
              <item row="0" column="0">
               <widget class="QProgressBar" name="updProgressBar">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="noUpdateLabel">
                <property name="text">
                 <string>Нет доступных обновлений</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="startStopButton">
        <property name="text">
         <string> Начать установку </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="renewButton">
        <property name="text">
         <string> Обновить список </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="niniteOpenBtn">
        <property name="text">
         <string> Открыть ninite.com </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="drpOpenBtn">
        <property name="text">
         <string> Запустить драйверпак </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exitButton">
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="infoBox">
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="InfoLabel">
         <property name="text">
          <string>No info</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="hideButton">
           <property name="text">
            <string>Hide</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="showButton">
      <property name="text">
       <string>Show Info</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>566</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Наборы</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Галки</string>
    </property>
    <addaction name="markedAllAction"/>
    <addaction name="unmarkedAllAction"/>
    <addaction name="unmarkedInstalledAction"/>
    <addaction name="separator"/>
    <addaction name="markedSoftOnly"/>
    <addaction name="markedUpdatesOnly"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="separator"/>
    <addaction name="startSetupAction"/>
    <addaction name="rescanAction"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="markedAllAction">
   <property name="text">
    <string>Поставить все</string>
   </property>
  </action>
  <action name="unmarkedAllAction">
   <property name="text">
    <string>Снять все</string>
   </property>
  </action>
  <action name="startSetupAction">
   <property name="text">
    <string>Начать установку</string>
   </property>
  </action>
  <action name="rescanAction">
   <property name="text">
    <string>Пересмотреть папки</string>
   </property>
  </action>
  <action name="markedSoftOnly">
   <property name="text">
    <string>Кроме обновлений</string>
   </property>
  </action>
  <action name="markedUpdatesOnly">
   <property name="text">
    <string>Только обновления</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Справка</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="unmarkedInstalledAction">
   <property name="text">
    <string>Снять установленные</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>376</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
